.github/workflows/bencher.yml

```diff
@@ -132,6 +133,7 @@ jobs:
             --hash $(git rev-parse HEAD~1) \
             --branch-start-point main \
```

Comment:
Hm, looking at docs it says we need `--start-point`, but we only have `--branch-start-point`. This was renamed per https://bencher.dev/docs/reference/changelog/#v0421